mmv1/Gemfile.lock

```diff
@@ -1,57 +1,59 @@
 GEM
```

Comment:
Were these Gemfile.lock changes intentional? I don't think so, so they can probably be reverted! They're causing the build to fail.

Reply:
These changes were the result of the `bundler update`, reverted those.